examples/step-74/doc/results.dox

```diff
@@ -0,0 +1,195 @@
+solutions in vtu format.
+
+Convergence rate for the smooth case with polynomial degree 3:
+@code
+DEAL::Cycle 0
+DEAL::Number of active cells:       16
+DEAL::Number of degrees of freedom: 256
+   Solving system...
+DEAL::Writing solution to <sol_Q3-00.vtu>
+   Error in the L2 norm       :     0.00193285
+   Error in the H1 seminorm   :     0.106087
+   Error in the enery norm    :     0.150625
+
+DEAL::Cycle 1
+DEAL::Number of active cells:       64
+DEAL::Number of degrees of freedom: 1024
+   Solving system...
+DEAL::Writing solution to <sol_Q3-01.vtu>
+   Error in the L2 norm       :     9.60497e-05
+   Error in the H1 seminorm   :     0.0089954
+   Error in the enery norm    :     0.0113265
+
+DEAL::Cycle 2
+DEAL::Number of active cells:       256
+DEAL::Number of degrees of freedom: 4096
+   Solving system...
+DEAL::Writing solution to <sol_Q3-02.vtu>
+   Error in the L2 norm       :     5.60638e-06
+   Error in the H1 seminorm   :     0.000901791
+   Error in the enery norm    :     0.000973568
+
+DEAL::Cycle 3
+DEAL::Number of active cells:       1024
+DEAL::Number of degrees of freedom: 16384
+   Solving system...
+DEAL::Writing solution to <sol_Q3-03.vtu>
+   Error in the L2 norm       :     3.48399e-07
+   Error in the H1 seminorm   :     0.000107055
+   Error in the enery norm    :     0.000108757
+
+DEAL::Cycle 4
+DEAL::Number of active cells:       4096
+DEAL::Number of degrees of freedom: 65536
+   Solving system...
+DEAL::Writing solution to <sol_Q3-04.vtu>
+   Error in the L2 norm       :     2.17928e-08
+   Error in the H1 seminorm   :     1.32657e-05
+   Error in the enery norm    :     1.33065e-05
+
+DEAL::Cycle 5
+DEAL::Number of active cells:       16384
+DEAL::Number of degrees of freedom: 262144
+   Solving system...
+DEAL::Writing solution to <sol_Q3-05.vtu>
+   Error in the L2 norm       :     1.36274e-09
+   Error in the H1 seminorm   :     1.65576e-06
+   Error in the enery norm    :     1.65683e-06
+
+degree = 3
+| cycle | cells | dofs   | L2        | L2...red.rate.log2 | H1        | H1...red.rate.log2 | Energy    |
+| 0     | 16    | 256    | 1.933e-03 | -                  | 1.061e-01 | -                  | 1.506e-01 |
+| 1     | 64    | 1024   | 9.605e-05 | 4.33               | 8.995e-03 | 3.56               | 1.133e-02 |
+| 2     | 256   | 4096   | 5.606e-06 | 4.10               | 9.018e-04 | 3.32               | 9.736e-04 |
+| 3     | 1024  | 16384  | 3.484e-07 | 4.01               | 1.071e-04 | 3.07               | 1.088e-04 |
+| 4     | 4096  | 65536  | 2.179e-08 | 4.00               | 1.327e-05 | 3.01               | 1.331e-05 |
+| 5     | 16384 | 262144 | 1.363e-09 | 4.00               | 1.656e-06 | 3.00               | 1.657e-06 |
+@endcode
+Thearetically, for polynomial degree $p$, the order of convergece in $L_2$
+norm and $H_1$ seminorm should be $p+1$ and $p$, respectively. Our numerical
+results are in good agreement with the theories.
+
+Errors of the singular case with polynomial degree 3.
```

Comment:
Since we don't know what the test cases are, this is hard to understand. You should explain what this case is (and what the first test case is) in the introduction.